src/Phinx/Migration/Manager.php

```diff
@@ -768,6 +768,58 @@ public function setSeeds(array $seeds)
         return $this;
     }
 
+    /**
+     * Get seed dependencies instances from seed dependency array
+     *
+     * @param AbstractSeed $seed Seed
+     *
+     * @return AbstractSeed[]
+     */
+    protected function getSeedDependenciesInstances(AbstractSeed $seed)
+    {
+        if (empty($this->seeds)) {
+            return [];
+        }
```

Comment:
This check seems unnecessary. If `$this->seeds` is an empty array, I think this method is not called.